ray-operator/controllers/ray/rayservice_controller_unit_test.go

```diff
@@ -815,6 +823,14 @@ func TestReconcileRayCluster(t *testing.T) {
 			updateKubeRayVersion:    true,
 			kubeRayVersion:          "new-version",
 		},
+		// Test 7: Zero downtime upgrade is enabled, but is enabled through the RayServiceSpec
```

Comment:
can we add more tests? We have 4 combinations `enableZeroDowntime` (true, false) and `rayServiceUpgradeStrategy` (NewCluster, None).

Reply:
Added the following combinations:
env var: false, spec: true
env var: true, spec: false
env var: unset, spec: unset
env var: false, spec: false

Reply:
On second thought, I believe there are 6 cases based on `enableZeroDowntime` (true, false) and `rayServiceUpgradeStrategy` (NewCluster, None, "").

Reply:
I believe all 6 of the combinations are covered. The "" should be covered by the original tests and the ("NewCluster", "None") x (true, false) are covered by the new tests that are added.